hr_experience/views/hr_employee_view.xml

```diff
@@ -1,16 +1,17 @@
-<openerp>
-  <data>
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
     <!-- Add a menu to Human Resources to keep the curriculum. -->
     <menuitem id="menu_hr_experience" parent="hr.menu_hr_root" name="Experiences" sequence="0"/>
-    
+
     <!-- inheritEmployee -->
     <record model="ir.ui.view" id="view_employee_form_inherit">
         <field name="name">hr.experience.employee.form</field>
         <field name="model">hr.employee</field>
         <field name="inherit_id" ref="hr.view_employee_form"/>
         <field name="arch" type="xml">
             <notebook position="inside">
-                <page string="Academic Experiences" groups="base.group_user,base.group_hr_user">
+                <page string="Academic Experiences" groups="base.group_user,hr.group_hr_user">
```

Comment:
As 'hr.group_hr_user' has the base rights from 'base.group_user' it can be removed.